### PR TITLE
Add to Prisma Seed Script

### PR DESCRIPTION
**Pull request by ShouvikGhosh2048**
Add random data to the database when the seed script is run (#528).

### CODE REVIEW

**Comment by otobot1** [comment]
Looks like cross-env went read-only 2 years ago. We can use it for now, but I'd want to switch to an alternative soon.

**Comment by otobot1** [comment]
> Looks like cross-env went read-only 2 years ago. We can use it for now, but I'd want to switch to an alternative soonish.

It looks like environment variables *are* loaded when the seed script is run. `NODE_ENV` isn't set automatically by Next.js, but we can set it manually in `.env`. 

Instead of messing around with `NODE_ENV` though, let's use a new environment variable.

**Comment by ShouvikGhosh2048** [comment]
> > Looks like cross-env went read-only 2 years ago. We can use it for now, but I'd want to switch to an alternative soonish.
> 
> It looks like environment variables _are_ loaded when the seed script is run. `NODE_ENV` isn't set automatically by Next.js, but we can set it manually in `.env`.
> 
> Instead of messing around with `NODE_ENV` though, let's use a new environment variable.

@otobot1 I've changed the environment variable used. The PR is ready for review.

**Comment by otobot1** [comment]
@ShouvikGhosh2048 I had some time to go through the PR today. It looks good overall, but there are a couple of things to resolve before it will be ready for merging. I made some changes already (mostly formatting), please see the commit messages for details.

1. We need to seed the db with ~5 parent difficulties (the ones with a `parentDifficultyId` of 0) and 3-5 child difficulties per parent difficulty.

2. There's a few spots where an `!` is used to assert that something is defined. Please try to avoid these (and other TypesScript "escape hatches" whenever possible.
    * Try to resolve the undefined errors with explicit checks, or modify the function providing the value to narrow the return type. 
    * Using an "escape hatch" in the helper function is preferred over using it on the returned value. This way the programmer's knowledge of how the function works is handled in the function itself.
    * I spent a bit of time refactoring things to be more type safe, but I ran out of time today so I didn't merge them. They're stashed on [this branch](https://github.com/celestemods-com/CelesteMods/blob/prisma-seed/prisma/seed.ts).

**Comment by otobot1** [comment]
On second thought, we can leave the TypeScript changes as a followup issue. It doesn't need to be perfect, especially until v1.0 is out.

**Comment by ShouvikGhosh2048** [comment]
@otobot1 I've made the requested changes.

Does the `ts-node` part work for you?
I get the error `Error: Cannot find module '~/utils/randomValueGenerators'` when I run `npm run db-reset`.
I tried using `tsx` and that works.

**Comment by otobot1** [comment]
> @otobot1 I've made the requested changes.
> 
> Does the `ts-node` part work for you? I get the error `Error: Cannot find module '~/utils/randomValueGenerators'` when I run `npm run db-reset`. I tried using `tsx` and that works.

Oh that's my bad. I didn't test my changes enough. I'm seeing the same issue, and `tsx` seems like a good solution. If you have a chance, please make that change. Otherwise, I'll do it when I have a chance to do a more thorough review.

File: src/utils/randomValueGenerators.ts
Comment (otobot1) [inline src/utils/randomValueGenerators.ts:1]: Small stylistic changes.

**Pull request merged**